polaris-icons/icons/PasskeyMajor.yml

```
@@ -0,0 +1,13 @@
+name: Passkey
+set: major
+description: Used to represent a passkey
+keywords:
+  - passkey
+  - identity
+  - security
+authors:
+  - Ashley Allen
+version: 1
+date_added: 2022-11-15
+date_modified: 2022-11-15
+exclusive_use: Identity
```

These can be left blank unless there's a reason these should _only_ be used in identity

```suggestion
exclusive_use:
```

---

It does not have to be only for Identity. I will remove them.